bl-languages/fi_FI.json

```diff
@@ -0,0 +1,242 @@
+{
+	"language-data":
+	{
+		"native": "Suomi (Suomi)",
+		"english-name": "Finnish",
+		"last-update": "2016-06-11",
+		"author": "Tuomas K.",
+		"email": "",
+		"website": ""
+	},
+
+	"username": "Käyttäjätunnus",
+	"password": "Salasana",
+	"confirm-password": "Vahvista salasana",
+	"editor": "Kirjoittaja",
+	"dashboard": "Hallintapaneeli",
+	"role": "Tyyppi",
+	"post": "Blogikirjoitus",
+	"posts": "Blogikirjoitukset",
+	"users": "Käyttäjät",
+	"administrator": "Ylläpitäjä",
+	"add": "Lisää",
+	"cancel": "Peruuta",
+	"content": "Sisältö",
+	"title": "Otsikko",
+	"no-parent": "Ei ole",
+	"edit-page": "Muokkaa sivua",
+	"edit-post": "Muokkaa blogikirjoitusta",
+	"add-a-new-user": "Lisää uusi käyttäjä",
+	"parent": "Onko tämä jonkun sivun alasivu?",
+	"friendly-url": "Sivun osoite",
```

Comment:
"friendly-url": "Sievennetty URL",
(Roughly translates to reduced URLs but "sievä" is also pretty so that kind of references to pretty URLs :))